src/utils/getNormalizedStateData.js

```diff
@@ -30,6 +30,9 @@ import sortByYearAndMonth from "./sortByYearAndMonth";
  *   compared_to_month?: string
  *   value_change?: number
  *   percentage_change?: number
+ *   source_name: string
```

Comment:
We also want to update the `README` to thoroughly document the updated format of the data file.